pkg/sqlite/load.go

```diff
@@ -117,7 +117,7 @@ func (s *sqlLoader) addOperatorBundle(tx *sql.Tx, bundle *registry.Bundle) error
 	}
 
 	if _, err := addBundle.Exec(csvName, csvBytes, bundleBytes, bundleImage, version, skiprange, replaces, strings.Join(skips, ","), substitutesFor); err != nil {
-		return err
+		return fmt.Errorf("Failed to add bundle %q: %s", csvName, err.Error())
```

Comment:
Go recommends not capitalizing error message strings: https://github.com/golang/go/wiki/CodeReviewComments#error-strings